CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 
 🔥 *Features*
 * Secrets can now be used inside workflow functions
-
+* `sdk.secrets.get("name")` will now use passport-based authorization if `ORQUESTRA_PASSPORT_FILE` environment variable is set. Otherwise, passing a valid `config_name="..."` is required. Previously, "passport auth" was only used inside QE tasks; any other execution context required "config auth" regardless of environment variables.
```

Comment:
I'd leave out "Previously, "passport auth" was only used inside QE tasks; any other execution context required "config auth" regardless of environment variables." to avoid confusion. This wasn't a user facing thing and just explaining the new thing is enough for the change log. As long as this is captured inside the commit message, I think it's OK.

Reply:
Edited in 91e28e9